src/PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp

```diff
@@ -42,10 +42,12 @@ namespace ConstitutiveRelations {
  * E=\frac{9K\mu}{3K+\mu}=\frac{\mu(3\lambda+2\mu)}{\lambda+\mu}
  * \f] and the _Poisson ratio_ \f[
  * \nu=\frac{3K-2\mu}{2(3K+\mu)}=\frac{\lambda}{2(\lambda+\mu)}
- * \f]. Inversely, these relations read: \f[
- * K &=\frac{E}{3(1-2\nu)} \\
- * \lambda &=\frac{E\nu}{(1+\nu)(1-2\nu)} \\
- * \mu &=\frac{E}{2(1+\nu)}
+ * \f] 
+ * Inversely, these relations read: 
```

Comment:
Hi @tomwlodarczyk could you please add blank lines before and after the `\f[` `\f]` blocks? Clang-format likes to squash them together and then sometimes they don't render correctly. Thanks!